internal/app/singularity/verify.go

```diff
@@ -167,10 +179,28 @@ func (v verifier) getOpts(ctx context.Context, f *sif.FileImage) ([]integrity.Ve
 
 	// Add key material from certificate(s).
 	for _, c := range v.certs {
-		if err := verifyCertificate(c, v.intermediates, v.roots); err != nil {
+		// verify that the leaf certificate is not tampered and that is adequate for signing purposes.
+		chain, err := verifyCertificate(c, v.intermediates, v.roots)
+		if err != nil {
 			return nil, err
 		}
 
+		// Verify that the certificate is issued by a trustworthy CA (i.e the certificate chain is not revoked or expired).
+		if v.ocsp {
+			if len(chain) != 1 {
+				return nil, errors.New("Unhandled OCSP condition")
+			}
+
+			ocspErr := OCSPVerify(chain[0]...)
+			if ocspErr != nil {
+				// TODO: We need to decide whether this should be strict or permissive.
+				return nil, ocspErr
+			}
+
+			sylog.Infof("OCSP validation has passed")
```

Comment:
Should this be part of the verification message output that goes to STDOUT, rather than a log message to STDERR?

Another way of thinking about it... if someone uses `--quiet`... should this still appear?